modules/tensor_mechanics/src/materials/ADLAROMANCEStressUpdateBase.C

```diff
@@ -106,37 +119,84 @@ ADLAROMANCEStressUpdateBase::ADLAROMANCEStressUpdateBase(const InputParameters &
                            : NULL),
     _immobile_dislocation_increment(0.0),
     _immobile_old(0.0),
-    _stress_index(2),
+    _effective_strain_old(0.0),
+    _mobile_input_idx(0),
+    _immobile_input_idx(1),
```

Comment:
Switching these dislocation terms away from the mobile and immobile that I used in my original code and to the cell and wall dislocations that Laurent always uses in his code and presentations would be a good step for consistency

Reply:
Updated